### ISSUE TITLE
"No viral RdRP identified in input sequence (or a server error occured)"

### ISSUE THREAD

**Issue by batson** [serratus-bio/serratus.io#196]
**Describe the bug**
Timeout

Hash: 7603a13b7fcfe7768ded959684192115c6f8947c
Submission time (ISO): 2022-09-14T05:43:31.734Z

Submission was genome of Wuhan Mosquito Virus 6 from NCBI: https://www.ncbi.nlm.nih.gov/nuccore/MW434423.1?report=fasta

**Comment by ababaian** [comment]
Yes this is a known bug :) I'd have sent you this link for palmID for WMV6 otherwise. `palmscan` PSSM are not hitting Orthomyxoviridae very efficiently, it's a false-negative. You'll need to perform the SRA-list parsing manually against the micro-assembly `rdrp1.mu.fa` contigs. Once you have that you can walk through the remaining palmID R script to generate the report locally.

Sorry about that. I'll leave this open as a ticket to swing back to it.

**Comment by batson** [comment]
Thanks, @ababaian. I ended up going through all the WMV6 microassembly contigs more or less manually, and finding sequence from Tunesia, the Philippines, and CT (USA) which hadn't made it to GenBank. We're updating the paper accordingly (going from five continents to six!) and citing Serratus!
